examples/cache/Dockerfile-service

```diff
@@ -7,4 +7,7 @@ COPY ./start_service.sh /usr/local/bin/start_service.sh
 COPY ./service-envoy.yaml /etc/service-envoy.yaml
```

Comment:
no need for this either

Reply:
so generally you want to split Flask based service and envoy, right?

Reply:
yep